### PR TITLE
Remove DebounceInput from JsonSchema_string

### LINKED ISSUE #4745: Impossible to submit required params in react app which is using SwaggerUi component
### Prerequisites  
 - OS: macOS
 - Browser: chrome 
 - Version: 67
 - Method of installation: npm
 - Swagger-UI version: 3.17.3
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
**Swagger UI is used inside react component. React version is `^16.3.2`**

Example Swagger/OpenAPI definition: https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v3.0/petstore.yaml

Swagger-UI configuration options:
```js
SwaggerUI({
  dom_id: "#swagger",
  url: spec_url,
})
```

### Describe the bug you're encountering
Impossible to specify the required parameter for routes like `/get/{petId}`. The UI just remove it when I'm clicking on Try it Out -> Execute button. 

### To reproduce...

1. Click on any route with required parameter. E.g. /pet/{petId}
2. Click "Try it out" button
3. Specify pet id
4. Click on "execute" button
4. See error - pet id was removed, petId field is red. UI doesn't send any http request

### Expected behavior
UI sends the HTTP request and display the results

### Screenshots
[video](https://i.imgur.com/qixGxZn.gifv)

### Additional context or thoughts
This is most visible issue that have direct impact on functionality. However, there are tons of JS errors in the chrome console. They appear on almost every stage: page load, authorization modal window, submiting requests, etc.

I think most of problems come from different react versions used in app and swagger ui

Other related issues: textarea with console curl command always show `curl -X "undefined"` instead of actual command(HTTP request is different)

### PR DESCRIPTION
**Pull request by Trial-In-Error**
DebounceInput doesn't appear to work with React 16; removing it causes
the inputs to function in React 16. Additionally, it's the only input in
the project protected this way.

Allows string inputs to function with React 16. Fixes #4745.

Manually tested in an existing React 16 application by building this project and moving the resulting build artifacts to the application's `node_modules/swagger-ui/dist` folder.

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [x] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/core/json-schema-components.js [hunk 1]
```diff
@@ -87,19 +86,18 @@ export class JsonSchema_string extends Component {
...
                      className={ errors.length ? "invalid" : ""}
                      title={ errors.length ? errors : ""}
-                     value={value}
-                     minLength={0}
-                     debounceTimeout={350}
-                     placeholder={description}
+                     value={ value }
+                     minLength={ 0 }
+                     placeholder={ description }
                      onChange={ this.onChange }
```
Comment (MynockSpit) [inline src/core/json-schema-components.js]: If we need the same behaviour, we could change either this line to onChange={`_.debounce(this.onChange, 350)`} or onChange to `onChange = _.debounce(() => this.props.onChange(this.state.value))`.

**Comment by nkbt** [comment]
As you can observe https://codepen.io/nkbt/pen/VvmzLQ?editors=0010 it certainly works with the latest react no issues. The problem is with the code you have around it not with debounce + react 16 specifically.

File: src/core/json-schema-components.js (hunk 1 above)
Comment (shockey) [reply src/core/json-schema-components.js]: I'm pretty sure we do need the behavior - I'll need to look back into some old tickets to confirm.

**Comment by tim-lai** [comment]
@Trial-In-Error Thanks for the PR! Thanks for waiting so long for a response. React 16 introduces several breaking changes to SwaggerUI. It is on our roadmap to update to React 16, albeit no eta. However, as noted by @nkbt , I'm not convinced that DebounceInput is one of the breaking changes.

**Comment by MynockSpit** [comment]
It may not be the cause, but removing it certainly fixes the issue and hasn’t produced any other bugs. We’ve had this change in production since this PR was created. 

Do you have/know of an alternate fix? Otherwise it’d be a bit annoying that you’re willing to ignore a PR that fixes a bug a year(ish) old with no perceivable downsides.

**Pull request closed without merging**